runtime/fdlimits/fdlimits.go

```diff
@@ -0,0 +1,25 @@
+package fdlimits
+
+import (
+	"github.com/ethereum/go-ethereum/common/fdlimit"
```

Comment:
I see no downside to doing this by default, so :+1:. I looked at the go-ethereum lib because I was curious why we were adding this dependency on them. It looks like the benefit here is that they have the major os-specific variants set up with appropriate build tags, so that we can have this simple `Raise(Maximum()) ` code (including eg Windows where the raise is a no-op). seems fine.